doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+    ```sql
+    SELECT id FROM mz_sources WHERE name = 'mz_source';
+    ```
+
+1. Use the [`mz_source_statuses`](/sql/system-catalog/mz_internal/#mz_source_statuses) table to check the overall status of your source:
+
+    ```sql
+    SELECT * FROM mz_internal.mz_source_statuses
+    WHERE id = '<source_ID>';
+    ```
+
+    <p></p>
+
+    ```nofmt
+     id  |   name    |   type   |  last_status_change_at   | status  | error | details
+    -----+-----------+----------+--------------------------+---------+-------+---------
+     u61 | mz_source | postgres | 2023-05-24 18:12:35.7+00 | running |       |
+    (1 row)
+    ```
+
+    If `status` is `running`, move on to the next step.
+
+    However, if `status` is `stalled` or `failed`, there's likely a configuration issue for you to fix. Check the `error` field for details and fix the issue before moving on. Also, if `status` is `starting` for more than a few minutes, [contact support](/support/).
+
+2. Once you know the source is running, use the [`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table to check the status of the initial snapshot:
+
+    ```sql
+    SELECT bool_and(snapshot_committed)
+    FROM mz_internal.mz_source_statistics
+    WHERE id = '<source_ID>';
+    ```
+    <p></p>
+
+    ```noftm
+     snapshot_committed
+    --------------------
+     t
+    (1 row)
+    ```
+
+    Once `snapshot_commited` is `t`, move on to the next step. Snapshotting can take between a few minutes to several hours, depending on the size of your dataset and the size of the cluster replica you chose for your `ingest_rds_postgres` cluster.
+
+## Step 7. Right-size the cluster
+
+After the snapshotting phase, Materialize starts ingesting change events from the PostgreSQL replication stream. For this work, Materialize generally performs well with an `xsmall` replica, so in this step, you'll resize the replica in your source cluster.
+
+1. Still in the `psql` shell connected to Materialize, remove the `medium` replica from the source cluster:
+
+    ```sql
+    DROP CLUSTER REPLICA ingest_rds_postgres.r1;
+    ```
+
+1. Add an `xsmall` replica to the source cluster:
+
+    ```sql
+    CREATE CLUSTER REPLICA ingest_rds_postgres.r1 SIZE 'xsmall';
+    ```
+
+1. Use the [`SHOW CLUSTER REPLICAS`](/sql/show-cluster-replicas/) command to check the status of the new replica:
+
+    ```sql
+    SHOW CLUSTER REPLICAS WHERE cluster = 'ingest_rds_postgres';
+    ```
+    <p></p>
+
+    ```noftm
+           cluster       | replica |  size  | ready
+    ---------------------+---------+--------+-------
+     ingest_rds_postgres | r1      | xsmall | t
+    (1 row)
+    ```
+
+1. Going forward, you can use the [`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table to verify that your new replica size is sufficient for Materialize to keep up with the change events coming through the replication stream:
```

Comment:
RDS has a metric for this called "Oldest Replication Slot Lag" which is exactly what we want-- @bobbyiliev can you help Jesse give folks instructions on how to find this, as you've shown in https://github.com/MaterializeInc/materialize/issues/19142 ?